scudo-whitepaper.md

```diff
@@ -163,7 +163,9 @@ Once the Director repository decides which images are required and signs targets
 
-Performing full in-toto verification can be a resource intensive process for some vehicle ECUs. As a stopgap measure, the Image repository may perform in-toto verification to generate a new “summary link” metadata file. The link can be used as proof that the Image repository performed the complete verification workflow, and the repository can then send this summary link metadata to the vehicle. Using this lightweight verification workflow requires more trust in the Image repository, as it is performing the verification rather than the client, which in this case would be the primary ECU. Further, in these situations, the Image repository could become a single point of failure for thousands of vehicles. So, while this approach can significantly reduce the metadata and verification overhead, it must only be used when full verification is categorically impossible on the vehicle.
+Performing full in-toto verification can be a resource intensive process for some vehicle ECUs, such as those designated as partial verification ECUs in Uptane. In these situations, other ECUs capable of performing in-toto verification are expected to step in for the the ECUs that cannot verify for themselves. This responsibility could fall to the primary ECU or a collection of full verification secondaries that share the extra load on behalf of partial verification secondary ECUs.
+
+For vehicles with no ECUs capable of performing in-toto verification, as a stopgap measure, the Image repository may perform in-toto verification to generate a new “summary link” metadata file. The link can be used as proof that the Image repository performed the complete verification workflow, and the repository can then send this summary link metadata to the vehicle. Using this lightweight verification workflow requires more trust in the Image repository, as it is performing the verification rather than the client, which in this case would be the primary ECU. Further, in these situations, the Image repository could become a single point of failure for thousands of vehicles. So, while this approach can significantly reduce the metadata and verification overhead, it must only be used when full verification is categorically impossible on the vehicle.
 
 Scudo can clearly combine the state of the art in software update security and software supply chain security systems. However, as with any framework that relies on user-defined policies, Scudo also requires that its policies be designed with secure defaults, allowing each part of the software supply chain as much privilege as is absolutely necessary.
```

Comment:
Add the word "only" to the last sentence  i.e. "allowing each part of the software supply chain only as much..."